packages/javascript-sdk/src/config/index.ts

```diff
@@ -28,6 +28,7 @@ function setDefaults(options: ConfigOptions): ConfigOptions {
     ...options,
     oauthThreshold: options.oauthThreshold || DEFAULT_OAUTH_THRESHOLD,
     logLevel: options.logLevel || 'none',
+    platformHeader: options.platformHeader || false,
```

Comment:
Is this what you were referring to, @ryanbas21?

Reply:
yup!